girder_worker/docker/io/__init__.py

```diff
@@ -0,0 +1,397 @@
+import os
+import errno
+import stat
+import abc
+import httplib
+import urlparse
+import ssl
+import sys
+import six
+
+
+@six.add_metaclass(abc.ABCMeta)
+class StreamConnector(object):
```

Comment:
Do all the classes in this module, and their descendants, belong in the `docker` package? I see that some of them reference the notion of container arguments, but a lot of this functionality looks more generic, at least on the surface.

Reply:
I think the names are more generic than they really are, they belong in the docker package, I will consider a rename.